source/Server.Extensibility/HostServices/Model/PackageReference.cs

```diff
@@ -129,7 +129,7 @@ public PackageReference()
         public string PackageId { get; set; } = string.Empty;
 
         /// <summary>
-        /// Feed ID, name or a variable-expression
+        ///     Feed ID, name or a variable-expression
```

Comment:
Why the extra spaces? :)

Reply:
Just rolling with the existing convention in the file 😆 No idea.